software/core/oqm-core-characteristics/app/UIs.py

```
@@ -0,0 +1,177 @@
+		output.plugin = sorted(output.plugin, key=lambda c: c.order)
+		
+		cls.ui_cache = output
+		
+		return output
+	
+	@classmethod
+	def get_uis_return(cls) -> UisResponse:
+		return cls.get_uis_cache().to_response()
+	
+	@classmethod
+	def get_ui_icon(cls, category: str, id: str) -> StreamingResponse:
```

Ruff (0.15.12)

[error] Function argument `id` is shadowing a Python builtin

(A002)